sdk/core/Azure.Core/README.md

```diff
@@ -82,9 +83,25 @@ SecretClient client = new SecretClient(new Uri("http://example.com"), new Defaul
 
 More on client configuration in [client configuration samples](https://github.com/Azure/azure-sdk-for-net/blob/main/sdk/core/Azure.Core/samples/Configuration.md)
 
+### Customzing Request Using `RequestContext`
+
+Besides general configuration of _serice client_ through `ClientOptions`, you can customize each individual request sent by _service client_
+using `RequestContext`.
+
+_Service clients_ have methods that can be used to call Azure services. We refer to these client methods _service methods_. You can
+pass `RequestContext` to each _service method_ as a parameter.
+
+```C# Snippet:SetRequestContext
+Response response = await client.GetPetAsync("snoopy", new RequestContext()
+{
+    ErrorOptions = ErrorOptions.Default
+});
+```
+
+More on request customization in [RequestContext samples](https://github.com/Azure/azure-sdk-for-net/blob/main/sdk/core/Azure.Core/samples/RequestContext.md)
```

Comment:
This doc is outdated and needs to be updated, specifically this section: https://github.com/Azure/azure-sdk-for-net/blob/main/sdk/core/Azure.Core/samples/ProtocolMethods.md#using-requestoptions-to-customize-behavior

Reply:
updated

Reply:
Thank you!